### ISSUE TITLE
Other stage script formatting options

### ISSUE THREAD

**Issue by severdia** [nyousefi/Fountain#20]
It would be great to have these stage script formatting options:

Scene: The scene location.
Time: When the scene takes place
Setting: A description of the scene
At Rise: What is happening as the scene begins

Below is an example of the formatting of each of these:

![screen shot 2016-11-09 at 5 30 33 pm](https://cloud.githubusercontent.com/assets/355924/20161313/581d57de-a6a2-11e6-80e2-87de1a9b68e2.png)

![screen shot 2016-11-09 at 5 31 06 pm](https://cloud.githubusercontent.com/assets/355924/20161315/5b6635f0-a6a2-11e6-94e9-68da2793f34a.png)

**Comment by FibreFoX** [comment]
Just being curious, how does this visually differ from issue #15 ? (like a table without drawn lines)

**Comment by severdia** [comment]
A little. The characters would be formatted more like this:

![screen shot 2016-11-09 at 5 53 07 pm](https://cloud.githubusercontent.com/assets/355924/20161747/7dd6ccbe-a6a5-11e6-9e15-9a3925a2454d.png)

**Comment by FibreFoX** [comment]
So the main difference is (just) the underlining?

**Comment by severdia** [comment]
And the colons, which I pointed out in that other thread as a potential problem with using those as delimiters.

**Comment by fuanegua** [comment]
Yes, that would be great. I miss it as well!

**Comment by Reaper10** [comment]
you forgot stage directions needs to be added the syntax
it could be
+stage directions+
https://www.youtube.com/watch?v=QnTOxfu2RXw&t=82s

**Comment by shaedrich** [comment]
@severdia Can't you use [synopses](https://fountain.io/syntax/#sections-synopses) syntax or do you need it to be more metadata-style? In that case, maybe you could utilize a [frontmatter](https://daily-dev-tips.com/posts/what-exactly-is-frontmatter/)
@severdia Couldn't you use a table for dramatis personae?
@Reaper10 Would be cool if markdown added `text-align:right` syntax so Fountain could just use this. For the time being, you could misuse [transitions](https://fountain.io/syntax/#transition) for the time being

**Comment by severdia** [comment]
@shaedrich 

> @severdia Can't you use [synopses](https://fountain.io/syntax/#sections-synopses) syntax or do you need it to be more metadata-style? In that case, maybe you could utilize a [frontmatter](https://daily-dev-tips.com/posts/what-exactly-is-frontmatter/)

This is not front matter or synopses. It's typically act or scene metadata. 

> @severdia Couldn't you use a table for dramatis personae?

Yes, that's what I was proposing, but there was no response, so I implemented what was discussed in the other issue for my own purposes.

**Comment by shaedrich** [comment]
@severdia But front matter is for storing metadata. That's why I suggested it. \
Is this implementation public?

**Comment by severdia** [comment]
@shaedrich But it isn't front matter. The implementation is in this app: https://apps.apple.com/us/app/scriptigo-pro/id1444743519

**Comment by shaedrich** [comment]
@severdia I'm not talking about dramatis personae, I was talking about this: 

> Scene: The scene location.
> Time: When the scene takes place
> Setting: A description of the scene
> At Rise: What is happening as the scene begins

Yeah, it's in the app but it's non-standard. Nobody prevents you from using this app but it's not implemented into Fountain (yet). I'm just trying to help and bounce off some ideas

**Comment by severdia** [comment]
@shaedrich Yes, those items are not front matter. They could be at the beginning of every scene in the script. Front matter is literally the front of the script (typically first page only).

**Comment by shaedrich** [comment]
@severdia Does it have to be? I mean, it's essentially a block that isn't parsed by Markdown but YAML containing key-value pairs

**Comment by severdia** [comment]
Yes, they're essentially key-value pairs. Yes, they're needed at the beginning of each scene. I didn't create the format. It's a convention.
